boot/zephyr/boards/mimxrt1060_evkb.conf

```diff
@@ -0,0 +1,5 @@
+# Copyright 2021-2022 NXP
```

Comment:
I suppose you could as well change this "Copyright" if you want to...

Reply:
I didn't know what to do in this case. I don't represent NXP and I was merely copying the configuration file from the mimxrt1060_evk as recommended by one of the NXP engineers. So I decided not to make any changes to the file.